### ISSUE TITLE
Change stress test runs to use "createdump"

### ISSUE THREAD

**Issue by mikem8361** [dotnet/coreclr#12823]
Probably mostly script work enabling dump creation with the proper environment variable, etc. It is currently Linux only.

**Comment by lt72** [comment]
@mikem8361: is this still relevant after recent dump debugging improvements? if not please close.

**Comment by mikem8361** [comment]
Yes it is.  We should change dumpling to use createdump for Linux to better test createdump

**Comment by lt72** [comment]
@DrewScoggins Drew, do you think you can act on this one anytime soon?
@danmosemsft @mikem8361 FYI

**Comment by mikem8361** [comment]
Dan and I are meeting on Wednesday to discuss this and other dumpling/symbol client/createdump things.

**Comment by sillyousu** [comment]
>Yes it is. We should change dumpling to use createdump for Linux to better test createdump

any updates?

`createdump` (2.0, 2.1, 2.2) never ever work for our Linux app.

**Comment by mikem8361** [comment]
This issue is about our internal stress testing run and it hasn't been changed.

Can you open another issue about the problem you are having with createdump and your Linux app which all the details?
